README.md

```diff
@@ -1,17 +1,33 @@
-# simple-cache-rs
-A simple im-mem cache for Rust
-
-![Docs](https://docs.rs/simple-cache-rs/badge.svg)
-[![Apache-2 licensed](https://img.shields.io/crates/l/simple-cache-rs.svg)](./LICENSE)
-[![CI](https://github.com/calvinbrown085/simple-cache-rs/workflows/Rust/badge.svg)](https://github.com/calvinbrown085/simple-cache-rs/actions?query=workflow%3ARust)
+# A simple in-mem cache for Rust
+[![Docs of original repo](https://docs.rs/simple-cache-rs/badge.svg)](https://docs.rs/simple-cache-rs/)
+[![Apache-2 licensed](https://img.shields.io/crates/l/simple-cache-rs.svg)](https://github.com/giuseppe998e/simple-cache-rs/blob/main/LICENSE)
+[![CI](https://img.shields.io/github/workflow/status/giuseppe998e/simple-cache-rs/rust-unit-test)](https://github.com/giuseppe998e/simple-cache-rs/actions?query=workflow%3ARust)
```

Comment:
Can we change these links back?

Reply:
"No problem, your repository, your rules." -cit.

Reply:
@giuseppe998e will you grab these? I don't think I can update these in your PR.